VideoLocker/src/main/java/org/edx/mobile/module/notification/BaseNotificationPayload.java

```diff
@@ -6,13 +6,31 @@
  * Created by hanning on 4/17/15.
  */
 public abstract class BaseNotificationPayload {
-    private @SerializedName("payload_version") String payloadVersion;
+    private @SerializedName("action") String action;
+    private @SerializedName("push_hash") String pushHash;
+    private @SerializedName("notification-id") String notificationId;
 
-    public String getPayloadVersion() {
-        return payloadVersion;
```

Comment:
Oh, we never put in a version did we. Do we still want to do that? cc @nasthagiri